### DISCUSSION TITLE
I can't find the stuff I need when I log in

### DISCUSSION THREAD

**Discussion by Tylersuard** [community/community#62326] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello.  When I open up my Github, I want to see my most recently visited or used repos on the upper left, but for some reason it appears to be just a random smattering with no order at all.  Also the notifications I receive in my email about someone responding to my issues on their repo, I would like to see those on my main page.  Right now I have no way of accessing them from the Github interface, at least that I can find.

**Comment by diyaryvzz** [comment]
You should be seeing it on your Github homepage, can you check a little more carefully?

**Comment by Tylersuard** [comment]
The link I clicked on said "send feedback", so that is what I am doing.
The dashboard has 3 different vertical sections:
Top Repositories(which I don't need, I need most recently accessed repositories)
A feed (which I don't need, none of the information is relevant to me.  I don't care who starred what repo)
The changelog and recommended repos.  Again, I don't need these.  I don't care about Github's changelog.

What I need is:
Panel on the left showing my own repos that I modified most recently
A second panel showing repos I visited most recently
The middle panel should show my notifications, like I get in my email, when someone responds to an issue I created.  Those are important, because I often can't proceed with a coding project until those issues get resolved.
The right panel could be repo recommender if it is a really good one.

These are all just suggestions, I am just offering feedback on the current dashboard.
